_posts/0000-01-02-Jitturohtaki.md

```diff
@@ -0,0 +1,11 @@
+---
+
+layout: slide
+
+title: "Welcome to our second slide!"
+
+---
+
+Your text
+
+Use the left arrow to go back!
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown